pocs/camera/sbig.py

```diff
@@ -106,7 +111,7 @@ def connect(self, set_point=None):
         else:
             self.filter_type = 'M'
 
-    def take_observation(self, observation, headers, **kwargs):
+    def take_observation(self, observation, headers=None, filename=None, *args, **kwargs):
```

Comment:
args is missing from docstring. And why was it added? It doesn't appear to be used.

Reply:
I get in the habit of adding `*args, **kwargs` to any method that might be overridden. Not sure that is a great idea but I've had to go back and add them more than I've ever had to remove them. I've looked for recommendations or best practices about this and haven't found anything.

I don't think there was a specific need here.